packages/entity-browser/src/routes/detail/components/DetailForm/formBuilder.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import {transformFieldName} from '../../../../util/detailView/reduxForm'
+import {Field} from 'redux-form'
+import {getFieldId} from '../../../../util/detailView/helpers'
+import ReduxFormFieldAdapter from '../ReduxFormFieldAdapter'
+import _get from 'lodash/get'
+import _startsWith from 'lodash/startsWith'
+import {LayoutBox} from 'tocco-ui'
+
+const layoutTypeNamespace = 'ch.tocco.nice2.model.form.components.layout.'
+const actionTypeNamespace = 'ch.tocco.nice2.model.form.components.action.'
+
+export default (entity, model, formName, formDefinition, formValues, formFieldUtils) => {
+  const isReadOnlyForm = () => formDefinition.displayType === 'READONLY'
```

Comment:
why function? could simply be replaced with `const readOnlyForm = formDefinition.displayType === 'READONLY'`